lib/esp/resources/custom_signature.rb

```diff
@@ -3,103 +3,94 @@ class CustomSignature < ESP::Resource
     #
-    # +include+ | The list of associated objects to return on the initial request.
+    #     +include+ | The list of associated objects to return on the initial request.
     #
-    # ===== Valid Includable Associations
+    #     ===== Valid Includable Associations
     #
-    # See {API documentation}[http://api-docs.evident.io?ruby#custom-signature-attributes] for valid arguments
-    #
-    # :call-seq:
-    #  find(id, options = {})
-
-    # :singleton-method: all
-    # Return a paginated CustomSignature list
+    #     See {API documentation}[http://api-docs.evident.io?ruby#custom-signature-attributes] for valid arguments
+    #   @return [ESP::CustomSignature]
 
-    # :singleton-method: create
-    # Create a CustomSignature
-    # :call-seq:
-    #   create(attributes={})
-    #
-    # ==== Parameter
-    #
-    # +attributes+ | Required | A hash of custom signature attributes
+    # @!method self.all
+    #   Return a paginated CustomSignature list
     #
-    # ===== Valid Attributes
-    #
-    # See {API documentation}[http://api-docs.evident.io?ruby#custom-signature-create] for valid arguments
+    #   @return [ActiveResource::PaginatedCollection<ESP::CustomSignature>]
+
+    # @!method self.create(attributes = {})
+    #   Create a CustomSignature
+    #   *call-seq* -> +super.create(attributes={})+
     #
-    # ==== Example
+    #   @param attributes [Hash] Required hash of custom signature attributes.
+    #     ===== Valid Attributes
     #
-    #  custom_signature = ESP::CustomSignature.create(description: "A test custom signature.", identifier: "AWS::IAM::001", name: "Test Signature", risk_level: "Medium")
+    #     See {API documentation}[http://api-docs.evident.io?ruby#custom-signature-create] for valid arguments
+    #   @return [ESP::CustomSignature]
+    #   @example
+    #     custom_signature = ESP::CustomSignature.create(description: "A test custom signature.", identifier: "AWS::IAM::001", name: "Test Signature", risk_level: "Medium")
 
-    # :method: save
-    # Create or update a CustomSignature
+    # @method save
+    #   Create or update a CustomSignature
     #
-    # ===== Valid Attributes
+    #   ===== Valid Attributes
     #
-    # See {API documentation}[http://api-docs.evident.io?ruby#custom-signature-create] for valid arguments
+    #   See {API documentation}[http://api-docs.evident.io?ruby#custom-signature-create] for valid arguments
     #
-    # ==== Example
-    #
-    #  custom_signature = ESP::CustomSignature.new(description: "A test custom signature.", identifier: "AWS::IAM::001", name: "Test Signature", risk_level: "Medium")
-    #  custom_signature.save
+    #   @return [Boolean]
+    #   @example
+    #    custom_signature = ESP::CustomSignature.new(description: "A test custom signature.", identifier: "AWS::IAM::001", name: "Test Signature", risk_level: "Medium")
+    #    custom_signature.save
 
-    # :method: destroy
-    # Delete a CustomSignature
+    # @method destroy
```

Comment:
Looks like yard is still parsing this as expected, but `@metho` is not listed as a valid tag.  Should be `@!method`.